datalad/support/gitrepo.py

```diff
@@ -329,19 +329,19 @@ def check_git_configured():
 
     Raises
     ------
-    RuntimeError  if any of those two ariables are not set
+    RuntimeError if any of those two variables are not set
     """
 
     check_runner = GitRunner()
     for c in 'user.name', 'user.email':
         try:
-            check_runner.run(['git', 'config', '--global', c])
+            check_runner.run(['git', 'config', c])
```

Comment:
Because datalad operates on paths which might be not under current repo,  we would then need to do this check for every GitRepo instance... Might get expensive?